modules/launcher/pom.xml

```diff
@@ -52,6 +52,11 @@
             <artifactId>mule-module-reboot</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.mulesoft.mule.runtime.modules</groupId>
+            <artifactId>mule-module-cluster-ee</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Careful with this, because this repository (mule) is part of the Mule CE (Community Edition), and here you would be adding a dependency on something that belongs to the Mule EE (Enterprise Edition) product.

Reply:
Ok I dont really need this, need some constants, I will redefine those